src/lib/yaraparse.py

```diff
@@ -186,44 +235,63 @@ def flatten_regex_or_tree(unit: Any) -> Optional[List[bytes]]:
         return None
 
 
+def urisfy_regex_or_tree(
+    unit: Any, is_ascii: bool, is_wide: bool, is_nocase: bool,
+) -> Optional[UrsaExpression]:
+    or_strings = flatten_regex_or_tree(unit)
+    if or_strings and all(s is not None for s in or_strings):
```

Comment:
Nitpicking at this point, but could this be converted as:
```suggestion
    if or_strings and all(or_strings):
```
or do you think this can cause problems with empty/non-truthy strings?

Reply:
I've tested the feasible cases and it shouldn't be a problem. I've inherited this line from the code before, so hopefully it's not an oversight of some kind.